docs/content/en/tasks/migrate/_index.md

```diff
@@ -6,227 +6,245 @@ weight: 1
 
-## The environment
+For the rest of this document, we'll refer to the pre-existing datacenter as `dc1` and the newly created one running on K8ssandra-operator as `dc2`. 
 
-It's assumed that the Cassandra cluster is running in the same Kubernetes cluster in which K8ssandra will run. The Cassandra cluster may have been installed with another operator (such as [CassKop](https://github.com/Orange-OpenSource/casskop)), a Helm chart (such as [bitnami/cassandra](https://github.com/bitnami/charts/tree/master/bitnami/cassandra)), or by directly creating the YAML manfiests for the StatefulSets and any other objects that were created.
+## Pre-requisites
 
-{{% alert title="Tip" color="success" %}}
-See <https://thelastpickle.com/blog/2019/02/26/data-center-switch.html> for a thorough guide on how to migrate to a new datacenter.
-{{% /alert %}}
+Pod IP addresses must be routable between all Kubernetes clusters involved in the migration, as well as Cassandra nodes when migrating from a non Kubernetes setup.
 
-## Check the replication strategies of keyspaces
+## Install K8ssandra-operator
 
-Confirm that the user-defined keyspaces and each of the `system_auth`, `system_distributed`, and `system_traces` are using `NetworkTopologyStrategy`.
+First, install cert-manager, which will be used by cass-operator:
 
-{{% alert title="Tip" color="success" %}}
-The `system_*` keyspaces include `system_auth`, `system_schema`, `system_distributed`, and `system_traces`.
-{{% /alert %}}
+```bash
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+```
 
-It is generally recommended to use a replication factor of `3`. If your keyspaces currently use `SimpleStrategy` and assuming you have at least 3 Cassandra nodes, then you would change the replication factor for `system_auth` as follows:
+Using [kustomize.io](https://kustomize.io) or [helm](https://github.com/helm/helm), install k8ssandra-operator and cass-operator in a dedicated namespace (we’ll use k8ssandra-operator here):
 
-```cql
-ALTER KEYSPACE system_auth WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1': 3}
-```
+### Using Helm
 
-`dc1` is the name of the datacenter as defined in `cassandra-rackdc.properties`, which you can find in the `/etc/cassandra/` directory of your Cassandra installation.
+```
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
+helm repo update
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator --create-namespace
+```
 
+### Using Kustomize
 
-Changing the recommendation may result in a topology change; that is, a token ownership change. 
+```
+kustomize build github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane\?ref\=v1.0.1 | kubectl apply --server-side --force-conflicts -f -
```

Comment:
The latest tag is 1.1.1, why not use that?

Reply:
Sounds good. We need "something" to do such updates in a more automated way.